backend/src/main/scala/bloop/logging/BloopLogger.scala

```diff
@@ -147,7 +148,7 @@ object BloopLogger {
     sw.toString()
   }
 
-  private lazy val colorsRegex = "\u001b\\[[0-9;]*m".r
+  private lazy val colorsRegex = "(\u001b|\u2190)\\[[0-9;]*m".r
```

Comment:
Why is this change needed?

Reply:
I added it to remove unwanted ansi codes, which often contain '\u2190' rather than '\u001b' (Escape).  I wasn't able to figure out where they were coming from.
My command line tool to strip color codes required this regex to remove some of it.
However, perhaps it should be part of a fix for removing unwanted ansi codes, instead of putting it here.